modules/exploits/multi/http/wp_dnd_mul_file_rce.rb

```diff
@@ -0,0 +1,161 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Remote::HTTP::Wordpress
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Wordpress Drag and Drop Multi File Uploader RCE',
+        'Description' => %q{
+          This module exploits a file upload feature of Drag and Drop Multi File
+          Upload - Contact Form 7.  However, the allowed file extension list can
```

Comment:
`However` doesn't fit here. Also, the module description should describe the vulnerable version like the documentation does.